packages/beeq/src/components/input/_storybook/bq-search.input.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { html, nothing } from 'lit-html';
+
+import meta from './bq-input.stories';
+
+const metaSearch: Meta = { ...meta, title: 'Components/Search' };
+export default metaSearch;
```

Comment:
Maybe we can be more explicit in this way, and keep consistency

```suggestion
import { default as metaInput } from './bq-input.stories';

const meta: Meta = { ...metaInput, title: 'Components/Search' };
export default meta;
```